test/specs/mainnet/01_TransferringFunds/Arbitrum.spec.js

```diff
@@ -660,247 +667,294 @@ describe("The SDK, when transfer a token with arbitrum network on the MainNet",
         );
-      } else {
+      } catch (e) {
         console.error(e);
-        assert.fail(
-          "The expected validation is not displayed when entered the very small value while adding the transactions to the batch."
+      }
+
+      try {
+        assert.isNotEmpty(
+          op.callData,
+          "The callData value is empty in the sign transactions added to the batch response."
         );
+      } catch (e) {
+        console.error(e);
       }
-    }
-  });
 
-  it("Regression: Perform the transfer native token with the exceeded Value while sign the added transactions to the batch on the arbitrum network", async () => {
-    // clear the transaction batch
-    try {
-      await arbitrumMainNetSdk.clearUserOpsFromBatch();
-    } catch (e) {
-      console.error(e);
+      try {
+        assert.isNotEmpty(
+          op.callGasLimit._hex,
+          "The hex value of the callGasLimit is empty in the sign transactions added to the batch response."
+        );
+      } catch (e) {
+        console.error(e);
+      }
+
+      try {
+        assert.isTrue(
+          op.callGasLimit._isBigNumber,
+          "The isBigNumber value of the callGasLimit is false in the sign transactions added to the batch response."
+        );
+      } catch (e) {
+        console.error(e);
+      }
+
+      try {
+        assert.isNotEmpty(
+          op.verificationGasLimit._hex,
+          "The hex value of the verificationGasLimit is empty in the sign transactions added to the batch response."
+        );
+      } catch (e) {
+        console.error(e);
+      }
+
+      try {
+        assert.isTrue(
+          op.verificationGasLimit._isBigNumber,
+          "The isBigNumber value of the verificationGasLimit is false in the sign transactions added to the batch response."
+        );
+      } catch (e) {
+        console.error(e);
+      }
+
+      try {
+        assert.isNotEmpty(
+          op.maxFeePerGas._hex,
+          "The hex value of the maxFeePerGas is empty in the sign transactions added to the batch response."
+        );
+      } catch (e) {
+        console.error(e);
+      }
+
+      try {
+        assert.isTrue(
+          op.maxFeePerGas._isBigNumber,
+          "The isBigNumber value of the maxFeePerGas is false in the sign transactions added to the batch response."
+        );
+      } catch (e) {
+        console.error(e);
+      }
+
+      try {
+        assert.isNotEmpty(
+          op.maxPriorityFeePerGas._hex,
+          "The hex value of the maxPriorityFeePerGas is empty in the sign transactions added to the batch response."
+        );
+      } catch (e) {
+        console.error(e);
+      }
+
+      try {
+        assert.isTrue(
+          op.maxPriorityFeePerGas._isBigNumber,
+          "The isBigNumber value of the maxPriorityFeePerGas is false in the sign transactions added to the batch response."
+        );
+      } catch (e) {
+        console.error(e);
+      }
+
+      try {
+        assert.isNotEmpty(
```

Comment:
paymasterAndData can be empty (0x) for those transactions which aren't sponsored.

Reply:
here empty means, missing value or null value. if we get a 0x value, then it will not consider an empty value.